turtle 1.3 breaks build

```
[2 of 2] Compiling Main             ( psc-package/Main.hs, dist/build/psc-package/psc-package-tmp/Main.o )

psc-package/Main.hs:142:15: error:
    • Couldn't match expected type ‘Line’ with actual type ‘Text’
    • In the first argument of ‘echo’, namely
        ‘("Package " <> pkg <> " does not exist in package set")’
      In a stmt of a 'do' block:
        echo ("Package " <> pkg <> " does not exist in package set")
      In the expression:
        do { echo ("Package " <> pkg <> " does not exist in package set");
             exit (ExitFailure 1) }

psc-package/Main.hs:153:9: error:
    • Couldn't match expected type ‘Line’ with actual type ‘Text’
    • In the first argument of ‘echo’, namely
        ‘("Updating " <> pack (show (length trans)) <> " packages...")’
      In a stmt of a 'do' block:
        echo ("Updating " <> pack (show (length trans)) <> " packages...")
      In the expression:
        do { getPackageSet config;
             db <- readPackageSet config;
             trans <- getTransitiveDeps db depends;
             echo ("Updating " <> pack (show (length trans)) <> " packages...");
             .... }

psc-package/Main.hs:155:11: error:
    • Couldn't match expected type ‘Line’ with actual type ‘Text’
    • In the first argument of ‘echo’, namely
        ‘("Updating " <> pkgName)’
      In a stmt of a 'do' block: echo ("Updating " <> pkgName)
      In the expression:
        do { echo ("Updating " <> pkgName);
             installOrUpdate config pkgName pkg }

psc-package/Main.hs:189:26: error:
    • Couldn't match type ‘Text’ with ‘Line’
      Expected type: [(Line, PackageInfo)]
        Actual type: [(Text, PackageInfo)]
    • In the second argument of ‘traverse_’, namely ‘trans’
      In a stmt of a 'do' block: traverse_ (echo . fst) trans
      In the expression:
        do { pkg@(PackageConfig {depends}) <- readPackageFile;
             db <- readPackageSet pkg;
             trans <- getTransitiveDeps db depends;
             traverse_ (echo . fst) trans }

psc-package/Main.hs:208:21: error:
    • Couldn't match type ‘Text’ with ‘Line’
      Expected type: Turtle.FilePath -> Line
        Actual type: Turtle.FilePath -> Text
    • In the second argument of ‘(.)’, namely ‘pathToTextUnsafe’
      In the first argument of ‘traverse_’, namely
        ‘(echo . pathToTextUnsafe)’
      In a stmt of a 'do' block:
        traverse_ (echo . pathToTextUnsafe) paths
```

--- a/purescript.cabal
+++ b/purescript.cabal
@@ -427,7 +427,7 @@ executable psc-package
                    optparse-applicative -any,
                    system-filepath -any,
                    text -any,
-                   turtle -any
+                   turtle <1.3
     main-is: Main.hs
     other-modules: Paths_purescript
     buildable: True

